tests/data/container_redis.json

```diff
@@ -0,0 +1,64 @@
+{
+	"metadata": {
+		"name": "podsandbox1-redis"
+	},
+	"image": {
+		"image": "docker://redis:latest"
```

Comment:
I don't really understand what this file is doing. Does it pull this docker redis image? If so, why the very specific redis versions below? Should it be pulling version `3.2.3` explicitly rather than `latest`?